views/event_details.hbs

```diff
@@ -5,7 +5,7 @@
   </div>
 </div>
 
-<div class="event-details text-center">
+<div id="event-details" class="event-details text-center">
```

Comment:
what's this for?

Reply:
yeah not sure when was this added but removed it cheers!